app/uk/gov/hmrc/gform/save4later/EncyryptedFormat.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.gform.save4later
+
+import play.api.libs.json._
+import uk.gov.hmrc.crypto.{ Crypted, CryptoWithKeysFromConfig, PlainText }
+
+import scala.util.{ Failure, Success, Try }
+
+object EncyryptedFormat {
+  def formatEncrypted[A](jsonCrypto: CryptoWithKeysFromConfig)(implicit format: Format[A]): Format[A] = new Format[A] {
```

Comment:
Why is this encryption needed?

Reply:
Form data may contain sensitive data. Its a good practice to encrypt data at REST. Save4later was doing this as well

Reply:
To enable data encryption was the reason why `save4later` was build. That was in the days mongo connection was not using https. But since https was introduced years ago and gform runs in protected zone I don't think we need to encrypt our data on top of all of that.

Reply:
Ok, as forms contains PII and mongo db is not password protected, someone with access to the protected zone can still access data in the DB. I will drop a message on #team-platsec and confirm if its needed